site/src/components/Workspace/Workspace.tsx

```diff
@@ -131,7 +131,7 @@ export const Workspace: FC<React.PropsWithChildren<WorkspaceProps>> = ({
 
         <WorkspaceStats workspace={workspace} handleUpdate={handleUpdate} />
 
-        {!!resources && !!resources.length && (
+        {Boolean(resources) && Boolean(resources.length) && (
```

Comment:
Not suggesting to do anything in this PR but `.length` might be something else we could bring up in FEV; my take on it is that we should always do `.length > 0` for these kinds of things since that seems more semantically accurate.

Reply:
Also seems CI is complaining about this.  I guess TypeScript is able to infer this is not undefined from `!` but not `Boolean`.

`Boolean(resources?.length)` maybe or `resources?.length > 0`.

Reply:
> my take on it is that we should always do .length > 0 for these kinds of things since that seems more semantically accurate.

I agree! Seems like something enforceable via ESLint: https://github.com/sindresorhus/eslint-plugin-unicorn/blob/main/docs/rules/explicit-length-check.md

I will add to FEV.